x-pack/filebeat/module/cisco/_meta/docs.asciidoc

```diff
@@ -98,6 +100,150 @@ The UDP port to listen for syslog traffic. Defaults to 9001.
 
 :fileset_ex!:
 
+[float]
+==== `ftd` fileset settings
+
+The Cisco FTD fileset primarily supports parsing IPv4 and IPv6 access list log
+messages similar to that of ASA devices as well as Security Event Syslog
+Messages for Intrusion, Connection, File and Malware events.
+
+*ECS Field mapping*
+
+The `ftd` fileset maps Security Event Syslog Messages to the Elastic Common
+Schema (ECS) format. The following table illustrates the mapping from
+Security Event fields to ECS. The `cisco.ftd` prefix is used when there is no
+corresponding ECS field available.
+
+Mappings for Intrusion events fields:
+[options="header"]
+|====================================
+| FTD Field | Mapped fields
+| ApplicationProtocol | network.protocol
+| DstIP | destination.ip
+| DstPort | destination.port
+| EgressInterface | cisco.ftd.destination_interface
+| GID | service.id
+| HTTPResponse | http.response.status_code
+| IngressInterface | cisco.ftd.source_interface
+| InlineResult | event.outcome
+| IntrusionPolicy | cisco.ftd.list_id
+| Message | message
+| Protocol | network.transport
+| SrcIP | source.ip
+| SrcPort | source.port
+| User | user.id, user.name
+| WebApplication | network.application
+|====================================
+
+Mappings for Connection and Security Intelligence events fields:
+[options="header"]
+|====================================
+| FTD Field | Mapped fields
+| ACPolicy | cisco.ftd.list_id
+| AccessControlRuleAction | event.outcome
+| AccessControlRuleName | cisco.ftd.list_id
```

Comment:
It was being mapped already to rule_name, thanks to a final rename processor in the pipeline. I did it that way to be able to quickly disable the rename when backporting to 7.4. However, I've fixed it already and put it in the CVS, because it had the side-effect of showing the wrong field name in the doc.